server/src/main/java/org/spine3/server/command/CommandBus.java

```diff
@@ -326,18 +277,36 @@ public void run() {
         return result;
     }
 
-    /** Obtains the instance of the {@link CommandStatusService} associated with this command bus. */
+    /**
+     * Obtains the instance of the {@link CommandStatusService} associated with this command bus.
+     */
     public CommandStatusService getCommandStatusService() {
         return commandStatusService;
     }
 
+    /* package */ boolean isMultitenant() {
```

Comment:
As long as we allow building `CommandBus` instances via public API, the `CommandBus.Builder` getters should be `public` as well. 

Otherwise, the users of the API will not be able to tell, what value the instance of `CommandBus.Builder` has.

In more details.

I imagine the `CommandBus.Builder` instance may be build in several code modules — in case of complex dev-staging-beta-producton deployment implementation. And the `Builder` instance may then be passed from one piece of code to another. And it is extremely helpful to understand its state. That's why the getters should be `public`.